test/services/messengerSync/rabbitmqMock.js

```diff
@@ -0,0 +1,22 @@
+let queues = {};
+
+const reset = () => {
+	queues = {};
+};
+
+const assertQueue = (queue, options) => {};
+
+const sendToQueue = (queue, messageBuffer, options) => {
+	if (!queues[queue]) queues[queue] = [];
+	queues[queue].push(messageBuffer);
+};
+
+const createChannel = async () => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected block statement surrounding arrow body; parenthesize the returned value and move it immediately after the `=>`.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5203703)